package.json

```
@@ -33,7 +33,7 @@
     "index.js"
   ],
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 18"
```

Note that the space here can be omitted.

```suggestion
    "node": ">=18"
```

---

sir, please.  that's my emotional support whitespace